src/scenario.jl

```diff
@@ -432,14 +432,16 @@ function run_model(domain::Domain, param_set::NamedDimsArray, corals::DataFrame,
     n_species::Int64 = domain.coral_growth.n_species
     n_groups::Int64 = domain.coral_growth.n_groups
 
-    # years to start seeding/shading
+    # years to start seeding/shading/fogging
     seed_start_year::Int64 = param_set("seed_year_start")
     shade_start_year::Int64 = param_set("shade_year_start")
+    fog_start_year::Int64 = param_set("fog_year_start")
 
     fogging::Real = param_set("fogging")  # percent reduction in bleaching mortality through fogging
```

Comment:
```suggestion
    fogging::Real = param_set("fogging")  # proportion of bleaching mortality reduction through fogging
```

Not caused by any changes in this PR but noticed this says "percent", implying values are $\in$ [0, 100].
Suggest changing to "proportion" to avoid this as values should be $\in$ [0, 1].